### DISCUSSION TITLE
Fetch large data

### DISCUSSION THREAD

**Discussion by samanxzrpor** [pgrimaud/instagram-user-feed#265] [General]
Hi
I used same pakage in past.

And when I send very request to instagram for fetch data ,It blokced my Ip or limited my Page 

So I want know if this package have these errors ??

**Comment by pgrimaud** [comment]
Hello,

That's Instagram limitation. The best way is to use proxies with a smart rotation.

You can find an example here: https://github.com/pgrimaud/instagram-user-feed/blob/master/examples/using-proxies.php
